Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM golang:alpine as build
+
+WORKDIR /build
+
+ADD ./ /build/
+ENV CGO_ENABLED=0
+ENV GOOS=linux
+
+RUN apk update && \
+	apk upgrade && \
+	apk add make gcc bash && \
+	rm -rf /var/cache/apk/*
+
+RUN cd ocis && make clean generate build
```

Comment:
Codacy found an issue: [Use WORKDIR to switch to a directory](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6290476)